libr/util/buf.c

```diff
@@ -205,16 +205,27 @@ R_API const ut8 *r_buf_buffer (RBuffer *b) {
 	return (b && !b->sparse)? b->buf: NULL;
 }
 
+static ut64 minimumZeroMaximum(ut64 limit, ut64 length, ut64 offset) {
```

Comment:
confusing function name

Reply:
Do you maybe have a suggestion for a better name?